Rewrite opening sentence (#2631)
theforeman/foreman-documentation: guides/common/modules/proc_creating-a-custom-product.adoc

[id="Creating_a_Custom_Product_{context}"]
= Creating a {customproducttitle}

> You can add repositories to a {customproduct} that you create using this procedure.

Reviewer: ```suggestion
Create a {customproduct} so that you can add repositories to the {customproduct}.
```
